src/shared/common/components/workstack.jsx

```diff
@@ -504,8 +520,8 @@ class WorkstackAllocate extends Component {
     }
 
     primaryCorrespondentSortStrategy(a, b, sortColumn) {
-        const aValue = sortColumn ? (this.getPrimaryCorrespondentDataValue(a, sortColumn.dataValueKey) || '').toUpperCase() : a.index;
```

Comment:
see comment under the method for this one please

Reply:
As above